docs/pt/docs/tutorial/body-multiple-params.md

```diff
@@ -0,0 +1,170 @@
+
+```JSON
+{
+    "item": {
+        "name": "Foo",
+        "description": "The pretender",
+        "price": 42.0,
+        "tax": 3.2
+    },
+    "user": {
+        "username": "dave",
+        "full_name": "Dave Grohl"
+    }
+}
+```
+
+!!! nota
+    Repare que mesmo que o `item` esteja declarado da mesma maneira que antes, agora é esperado que ele esteja dentro do corpo com uma chave `item`.
+
+
+**FastAPI** vai fazer a conversão automática a partir da requisição, assim esse parâmetro `item` receberá seu respectivo conteúdo específico e o mesmo ocorrerá com `user`.
+
+Ele vai realizar a validação dos dados compostos, e vai documentá-los de maneira compatível com `OpenAPI schema` e documentação automática.
+
+## Valores singulares no corpo
+
+Assim como existem uma `Query` e uma `Path` para definir dados adicionais para parâmetros de consulta e de rota, **FastAPI** provê o equivalente para `Body`.
+
+Por exemplo, extendendo o modelo anterior, você poder decidir por ter uma outra chave `importance` no mesmo corpo, além de `item` e `user`.
+
+Se você declará-lo como é, porque é um valor singular, **FastAPI** vai assumir que se trata de um parâmetro de consulta.
+
+Mas você pode instruir **FastAPI** para tratá-lo como outra chave do corpo usando `Body`:
+
+
+```Python hl_lines="23"
+{!../../../docs_src/body_multiple_params/tutorial003.py!}
+```
+
+Neste caso, **FastAPI** vai esperar um corpo como:
+
+
+```JSON
+{
+    "item": {
+        "name": "Foo",
+        "description": "The pretender",
+        "price": 42.0,
+        "tax": 3.2
+    },
+    "user": {
+        "username": "dave",
+        "full_name": "Dave Grohl"
+    },
+    "importance": 5
+}
+```
+
+Mais uma vez, ele vai converter os tipos de dados, validar, documentar, etc.
+
+## Múltiplos parâmetros de corpo e consulta
+
+Obviamente, você pode também declarar parâmetros de consulta assim que você precisar, de modo adicional a quaisquer parâmetros de corpo.
+
+Dado que, por padrão, valores singulares são interpretados como parâmetros de consulta, você não precisa explicitamente adicionar uma `Query`, você pode somente:
+
+```Python
+q: Optional[str] = None
+```
+
+como em:
+
+```Python hl_lines="28"
+{!../../../docs_src/body_multiple_params/tutorial004.py!}
+```
+
+!!! info "Informação"
+    `Body` também possui todas as validações adicionais e metadados de parâmetros como em `Query`,`Path` e outras que você verá depois.
+
+
+## Declare um único parâmetro de corpo indicando sua chave
+
+Suponha que você tem um único `item` parâmetro de corpo a partir de um modelo Pydantic `Item`.
+
+Por padrão, o **FastAPI** vai então esperar que seu conteúdo venha no corpo diretamente.
+
+Mas se você quiser que ele requisite por um JSON com uma chave `item` e dentro dele os conteúdos do modelo, como ocorre ao declarar vários parâmetros de corpo, você pode usar o parâmetro especial de `Body` chamado `embed`:
```

Comment:
```suggestion
Mas se você quiser que ele espere por um JSON com uma chave `item` e dentro dele os conteúdos do modelo, como ocorre ao declarar vários parâmetros de corpo, você pode usar o parâmetro especial de `Body` chamado `embed`:
```

Reply:
This one is tricky, is a matter of semantics:
It doesnt request anything, it expects to get a requests with...